devel/make/Makefile

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=make
+PKG_VERSION:=4.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=@GNU/make
+#PKG_SOURCE_URL:=https://ftp.gnu.org/gnu/make
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_VERSION:=$(PKG_VERSION)
+PKG_MD5SUM:=654f9117957e6fa6a1c49a8f08270ec9
+PKG_MAINTAINER:=Heinrich Schuchardt <xypron.glpk@gmx.de>
+PKG_LICENSE:=GPL-3.0+
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/make
+  SECTION:=devel
+  CATEGORY:=Development
+  TITLE:=make
+  DEPENDS:=
+  URL:=https://www.gnu.org/software/make/
+endef
+
+define Package/make/description
+  The Make package contains a tool to create executables from source files.
+endef
+
+TARGET_CFLAGS += $(FPIC) -Wno-unused-value
+
+CONFIGURE_ARGS += \
+	--host=$(REAL_GNU_TARGET_NAME) \
+	--target=$(REAL_GNU_TARGET_NAME) \
+	--prefix=/usr
+
```

Comment:
On 24.12.2015 08:18, Alexander Ryzhov wrote:
> In devel/make/Makefile
> <https://github.com/openwrt/packages/pull/2165#discussion_r48399725>:
> 
>> +  TITLE:=make
>> +  DEPENDS:=
>> +  URL:=https://www.gnu.org/software/make/
>> +endef
>> +
>> +define Package/make/description
>> +  The Make package contains a tool to create executables from source files.
>> +endef
>> +
>> +TARGET_CFLAGS += $(FPIC) -Wno-unused-value
>> +
>> +CONFIGURE_ARGS += \
>> +	--host=$(REAL_GNU_TARGET_NAME) \
>> +	--target=$(REAL_GNU_TARGET_NAME) \
>> +	--prefix=/usr
>> +
> 
> Not sure these customisations are necessary. At least, Entware-ng
> x86/arm/mipsel feeds are built without them.
> https://github.com/Entware-ng/entware-oldpackages-ports/blob/master/make/Makefile

You missed the second patch in my pull request which removed the
unnecessary lines.

Best regards

Heinrich Schuchardt